src/index.js

```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames';
 import isNegativeZero from 'is-negative-zero';
 import KeyCode from 'rc-util/lib/KeyCode';
 import InputHandler from './InputHandler';
+import omit from 'omit.js';
```

Comment:
is omit even used? I see you rely on the spread operator to catch the _rest_ of the props.

Reply:
Oops, it's my careless...But it has fixed in master. 
Anyway, thank you for your patience :)

Reply:
no porb, it didn't do any harm :)
we should probably setup some linting rules to prevent these kind of things from happening.